lib/protobuf/field/bool_field.rb

```diff
@@ -35,6 +35,19 @@ def decode(value)
         value == 1
       end
 
+      def define_decode_setter
+        field = self
+        field_name = field.name
+        tag_method_name = "_protobuf_decode_setter_#{field.tag}"
```

Comment:
Can we make this private, `Message.instance_eval { private tag_method_name.to_sym }` or some such?